### ISSUE TITLE
Adding assembled_file option assembly process

### ISSUE THREAD

**Issue by JohnMCMa** [Teichlab/tracer#77]
As we discussed in Teichlab/bracer#21 , when using 10X Genomics outputs, reads should be split by cell and then called their VDJ types using `bracer assembly`'s `--assembled_file` option. 

I suppose the main idea also applies to TraCeR, but `tracer assembly` doesn't have a similar function. This means I'll have to split the fastq, calculate fragment length and SD and run the entire VDJ assembly pipeline despite the cellranger vdj output files are already assembled. My rough calculation shows this increases the running time by 3-4X.

**Comment by mstubb** [comment]
Yes, this is a direct consequence of TraCeR being written prior to release of the 10x Genomics VDJ product. We’ll put this functionality on the to-do list but I’d be happy to accept a pull request that does this if you’d like to write one. 

All the best,

Mike
